lib/browser/api/web-contents.ts

```diff
@@ -482,7 +482,15 @@ WebContents.prototype._callWindowOpenHandler = function (event: Electron.Event,
   if (!this._windowOpenHandler) {
     return defaultResponse;
   }
-  const response = this._windowOpenHandler(details);
+
+  let response;
+  try {
+    response = this._windowOpenHandler(details);
+  } catch (e: any) {
+    event.preventDefault();
+    console.log(`Error in windowOpenHandler: ${e.message}`);
```

Comment:
This error will not be catchable by unhandled error handlers.  I'd prefer a different change where we instead handle `_callWindowOpenHandler` not returning 🤔 

E.g. Change the `const result = this._callWindowOpenHandler(event, details);` call later in this file to this:

```
let result;
try {
  result = this._callWindowOpenHandler(event, details);
} catch (err) {
  event.preventDefault();
  throw err;
}
```

That ensures that "error thrown in handler means no window opens" but also ensures the error is propagated to the global error handler

Reply:
Maybe I misunderstand, but so does mean that we'd still be hitting the uncaught exception handler if we crash inside of the callback, so this isn't actually fixing the issue and the release notes should be updated?